src/api.ts

```diff
@@ -11,6 +11,6 @@ function wait(delay: number) {
 export async function getPeople(): Promise<Person[]> {
   // keep this delay for testing purpose
   return wait(500)
```

Comment:
Consider handling potential errors in the promise chain to make the function more robust. You can add a catch block to handle any errors that might occur during the fetch operation.